client/src/pages/SignUp.js

```diff
@@ -109,65 +135,73 @@ const SignUp = () => {
-              <div>Job Seeker</div>
-            </span>
-            <span>
-              <input type="radio" value="recruiter" name="account_type" onClick={() => setAccountType("recruiter")} />
-              <div>Recruiter</div>
-            </span>
+          <span>
+            <input type="radio" value="seeker" name="account_type" onClick={() => setAccountType("seeker")} />
+            <div>Job Seeker</div>
+          </span>
+          <span>
+            <input type="radio" value="recruiter" name="account_type" onClick={() => setAccountType("recruiter")} />
+            <div>Recruiter</div>
+          </span>
           <h2>Email</h2>
-            <input type="email" value={email} onChange={(e) => setEmail(e.target.value)} placeholder="youremail@domain.com" />
+          <input type="email" value={email} onChange={(e) => setEmail(e.target.value)} placeholder="youremail@domain.com" />
           <h2>Password</h2>
-            <input type="text" value={password} onChange={(e) => setPassword(e.target.value)} placeholder="Password" />
+          <input type="text" value={password} onChange={(e) => setPassword(e.target.value)} placeholder="Password" />
           <h2>Preferred Industry</h2>
-            <select value={preferredIndustry} onChange={(e) => setPreferredIndustry(e.target.value)}>
-              <option value="accounting">Accounting</option>
-              <option value="education">Education</option>
-              <option value="information technology">Information Technology</option>
-              <option value="software development">Software Development</option>
-              {/* <option value=""></option> */}
-            </select>
+          <select value={preferredIndustry} onChange={(e) => setPreferredIndustry(e.target.value)}>
+            <option value="accounting">Accounting</option>
+            <option value="education">Education</option>
+            <option value="information technology">Information Technology</option>
+            <option value="software development">Software Development</option>
+            {/* <option value=""></option> */}
+          </select>
           <h2>Zip Code</h2>
-            <input type="text" value={zipCode} onChange={(e) => setZipCode(Number(e.target.value))} placeholder="Zip Code" />
-          {/* {fileUpload('resume')} */}
+          <input type="text" value={zipCode} onChange={(e) => setZipCode(Number(e.target.value))} placeholder="Zip Code" />
+          {uploaded
+          ? <p> Resume Uploaded </p>
+          : <>
+            <Button variant='contained' onClick={() => fileInputRef.current.click()}>
+              Upload A Resume
+            </Button>
+            <input onChange={handleUpload} multiple={false} ref={fileInputRef} type='file' hidden />
+            </>
+          }
           <button onClick={register} disabled={loading}>Create Account</button>
           <button onClick={register} disabled={loading}>Create Account</button>
```

Comment:
We should probably get rid of one of these buttons. Currently two are being rendered to the "seeker" view of the form